src/components/menu.svelte

```diff
@@ -42,6 +43,7 @@
       <ResumeMenu />
       <WorkMenu />
       <BasicsMenu />
+      <TagsMenu />
```

Comment:
I should figure out how how to make this component work much better. Forgetting to add this component cost me about 5 minutes figuring out what I was missing.